csrc/scheduler/pointwise.cpp

```diff
@@ -961,10 +970,10 @@ void schedulePointwise(Fusion* fusion, const PointwiseParams* pparams) {
     // Don't need to worry about view transformations, just merge reference tv
     // as we normally would.
 
-    std::unordered_map<int64_t, int64_t> reorder_map =
-        scheduler_utils::maybeReorderAsAllocationMap(reference_tv);
-    if (!reorder_map.empty()) {
-      reference_tv->reorder(reorder_map);
+    std::unordered_map<int64_t, int64_t> loop_reorder_map =
+        scheduler_utils::maybeReorderLoopAsAllocationMap(reference_tv);
+    if (!loop_reorder_map.empty()) {
```

Comment:
This and other `if-s` will go away with https://github.com/NVIDIA/Fuser/pull/5217.